nmr_commons/tracking/Trajectory.py

```diff
@@ -0,0 +1,58 @@
+import numpy as np
+import csv
+import copy
+
+
+class Trajectory:
+    """Trajectory class where trajectory is a list of point and each point is a tuple of two points in space"""
+    ####
+    # Constructor
+    def __init__(self, items=None, norm_items=None):
+        # self.list = items if items else []    #this was before, I worried about mutable parameters
+        self.list = copy.deepcopy(items) if items else []
+        # self.norm_list = norm_items if norm_items else []     #this was before, I worried about mutable parameters
+        self.norm_list = copy.deepcopy(norm_items) if norm_items else []
+        self.flag = None  # used in trajectories visualization,
+        # True = trajectory created from real peak, False = trajectory created from artifact
+
+    def __iter__(self):
+        return iter(self.list)
+
+    def __getitem__(self, item):
+        return self.list[item]
+
+    def __len__(self):
+        return len(self.list)
+
+    def add_point(self, point, position=None):
+        if position is not None:
+            self.list[position] = tuple(point)
+        else:
+            self.list.append(point)
```

Comment:
This method basically duplicates `list.insert` (https://docs.python.org/3.8/tutorial/datastructures.html) and latter can be used directly.